content/blog/modern-text-features/index.Rmd

```diff
@@ -0,0 +1,370 @@
+---
+output: hugodown::hugo_document
+
+slug: modern-text-features
+title: Modern Text Features in R
+date: 2021-02-06
+author: Thomas Lin Pedersen
+description: >
+  ragg has taken a major leap forward in text rendering capabilities with the
+  latest releases of systemfonts, textshaping, and ragg itself. This post will
+  go into detail with what is now possible and how it compares to the build in 
+  devices.
+
+photo:
+  url: https://unsplash.com/photos/bMybTSV7RFY
+  author: Natalia Y
+
+# one of: "deep-dive", "learn", "package", "programming", or "other"
+categories: [deep-dive] 
+tags: []
+---
+
+I'm extremely pleased to present some new functionality when it comes to text rendering and font support in R.
+This is the culmination of work that started during the development of the ragg package where I was first exposed to the intricacies of text rendering.
+The new features presented herein spans the systemfonts, textshaping, and ragg packages, but from a user point of view everything will be available simply by using the graphic devices in ragg.
+
+ragg is now in a state where it "just works", and works comparably across all major operating systems. 
+This is a first for a raster device in R and is the result of several years of dedicated work. 
+I couldn't be happier with this release!
```

Comment:
```suggestion
I'm extremely pleased to present the culmination of several years of work on font rendering, spanning the systemfonts, textshaping, and ragg packages.
The result of this work is that font rendering in the ragg graphics device now just works, regardless of what you throw at it. 

This includes:
```

(Need to link first use of package name to package websites)

Reply:
I thought we wanted to emphasise that this is the first device to perform reliably and consistently across operating systems?

Reply:
Do we link to package GitHub for packages that doesn't have pkgdown? Both systemfonts and textshaping are too under-the-hood to warrant a pkgdown site

Reply:
Oh yeah, maybe a last sentence like:

> This completes our efforts to create a high-quality, performant graphics device that works produces the same output on every operating system.

Yes, link to github for non-pkgdown sites.